### DISCUSSION TITLE
vless和docodemo是否支持了full-cone？

### DISCUSSION THREAD

**Discussion by universonic** [v2fly/v2ray-core#1843] [Q&A] [answered]
我看到socks协议新加入的vlite协议都已经支持了full-cone。我个人用vless比较多，因此想问下vless和docodemo是否已经支持了full-cone；如若没有，那是否有支持的计划呢？求大佬解答下，谢谢。

**Comment by AkinoKaede** [comment, marked as answer]
VLESS is supported.
